rpg_api/tests/pytest/endpoint/character_location/test_patch_character_locations.py

```diff
@@ -0,0 +1,246 @@
+import pytest
+from httpx import AsyncClient
+from fastapi import status
+from rpg_api.db.postgres.factory import factories
+from typing import Any
+from rpg_api.web.api.postgres.auth import auth_utils as utils
+from rpg_api.utils import dtos
+import uuid
+
+url = "/api/postgres/character-locations"
+
+
+def get_user_header(token: str) -> dict[str, Any]:
+    """Return access token for given data."""
+    return {"Authorization": f"Bearer {token}"}
+
+
+@pytest.mark.anyio
+@pytest.mark.parametrize(
+    "x, y",
+    [
+        # Int32 boundaries
+        (-2147483648, 0),
+        (2147483647, 0),
+        (0, -2147483648),
+        (0, 2147483647),
+        # Other values
+        (-2147483647, 0),
+        (2147483646, 0),
+        (0, -2147483647),
+        (0, 2147483646),
+        (0, 0),
+        (100, 100),
+        (-100, -100),
+    ],
+)
```

Comment:
I think we should store the max size of int32 in constants.py
```python
INT32_SIZE = 2**31 - 1
```

That way it will be easier to see what the test is doing, and to explain at the exam:

```suggestion
@pytest.mark.parametrize(
    "x, y",
    [
        # Int32 boundaries
        (-constants.INT32_SIZE + 1, 0),
        constants.INT32_SIZE, 0),
		...
    ],
)
```

Reply:
I did this, but I explicitly stated the values in the constants file:
INT32_MAX: int = 2147483647
INT32_MIN: int = -2147483648